react-components/stories/utilities/getAddModelOptionsFromUrl.ts

```diff
@@ -16,8 +16,8 @@ export function getAddModelOptionsFromUrl(localModelUrlFallback: string): AddMod
   }
 
   return {
-    modelId: -1,
-    revisionId: -1,
+    modelId: 3544114490298106,
```

Comment:
I don't really care, but I get negative PR reviews when I do this........ But good for testing.

Reply:
Ah, I added it by mistake :see_no_evil: Nice catch